Packs/PiHole/Integrations/PiHole/PiHole.yml

```diff
@@ -0,0 +1,220 @@
+commonfields:
+  id: PiHole
+  version: -1
+name: PiHole
+display: PiHole
+category: IT Services
+description: This is an integration to PiHole
+configuration:
+- display: Server URL (e.g. https://example.net)
+  name: url
+  type: 0
+  required: true
+  additionalinfo: Input the base url excluding the /admin/api.php part
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+- display: Auth Token
```

Comment:
please move this param to be below the url

Reply:
Done